package.json

```diff
@@ -85,15 +85,15 @@
   "sideEffects": false,
   "dependencies": {
     "@mswjs/cookies": "^0.2.2",
-    "@mswjs/interceptors": "^0.17.2",
+    "@mswjs/interceptors": "^0.17.5",
     "@open-draft/until": "^1.0.3",
     "@types/cookie": "^0.4.1",
     "@types/js-levenshtein": "^1.1.1",
     "chalk": "4.1.1",
     "chokidar": "^3.4.2",
     "cookie": "^0.4.2",
     "graphql": "^15.0.0 || ^16.0.0",
-    "headers-polyfill": "^3.0.4",
+    "headers-polyfill": "^3.1.0",
```

Comment:
We still have a dependency on `3.0` from `page-with` but I'm migrating from `page-with` in #1342. It doesn't affect runtime in any way, it's a testing setup thing.